src/notation/internal/notationplayback.cpp

```diff
@@ -716,7 +716,7 @@ QRect NotationPlayback::loopBoundaryRectByTick(LoopBoundaryType boundaryType, in
 
     for (int i = 0; i < score()->nstaves(); ++i) {
         Ms::SysStaff* ss = system->staff(i);
-        if (!ss->show() || !score()->staff(i)->show()) {
+        if (!ss || !ss->show() || !score()->staff(i)->show()) {
```

Comment:
this was to prevent a crash but is almost certainly hiding a bigger problem, will report separately